### PR TITLE
Back with jwt auth and Get / Post on workflows

### PR DESCRIPTION
**Pull request by GitBluub**

### CODE REVIEW

File: api/todo [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- toPassword check security
+- Separate User and UserDB
+- Craft JWT ?
+- Crud pipelines
```
Comment (Octopus773) [inline api/todo:4]: x

**Review by Octopus773 — CHANGES_REQUESTED** [review]
wesh wesh canne à peche

File: api/app/Main.hs [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
 
 main :: IO ()
-main = startApp
+main = do
+    key <- generateKey
+    let jwtCfg = defaultJWTSettings key
+    pool <- acquire (3, 1, connectionSettings)
+    run 8080 $ app jwtCfg $ State pool
+    where
+        connectionSettings = Connection.settings "localhost" 5432 "postgres" "password" "postgres"
```
Comment (AnonymusRaccoon) [inline api/app/Main.hs:28]: You probably should not hardcode that

**Review by Octopus773 — APPROVED** [review]

**Review by 0Nom4D — APPROVED** [review]

**Review by Arthi-chaud — APPROVED** [review]

**Review by Arthi-chaud — COMMENTED** [review]
The code is not documented

**Pull request merged**